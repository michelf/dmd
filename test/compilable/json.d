// PERMUTE_ARGS:
// REQUIRED_ARGS: -o- -X -Xftest_results/compilable/json.out
// POST_SCRIPT: compilable/extra-files/json-postscript.sh

struct X;

enum Y;
