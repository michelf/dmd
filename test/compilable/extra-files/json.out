[
{
"kind" : "module",
"file" : "compilable\/json.d",
"members" : [
{
"name" : "X",
"kind" : "struct",
"line" : 5}
,
{
"name" : "Y",
"kind" : "enum",
"line" : 7}
]
}
]
